version: '3.4'

services:

  bootnode:
    restart: on-failure:10
    image: pee/eth-dev
    container_name: eth-dev-boot
    networks:
      - ethdev
    environment:
      - RUN_BOOTNODE=true

#  devnode1:
#    restart: on-failure:10
#    image: pee/eth-dev
#    container_name: eth-dev-node1
#    networks:
#      - ethdev
#    environment:
#      - RUN_BOOTNODE=false

#  devnode2:
#    restart: on-failure:10
#    image: pee/eth-dev
#    container_name: eth-dev-node2
#    networks:
#      - ethdev
#    environment:
#      - RUN_BOOTNODE=false

#  miner:
#    restart: on-failure:10
#    image: pee/eth-dev
#    container_name: eth-dev-mine
#    networks:
#      - ethdev
#    environment:
#      - RUN_BOOTNODE=false

networks:
  ethdev:
    driver: bridge
    driver_opts:
      com.docker.network.enable_ipv6: "false"





